src/pages/bridge/Issue/IssueValidationSchema.ts

```diff
@@ -0,0 +1,13 @@
+import * as Yup from 'yup';
+import { IssueFormValues } from '.';
+import { transformNumber } from '../../../helpers/yup';
+
+export function getIssueValidationSchema(max: number) {
+  return Yup.object<IssueFormValues>().shape({
+    amount: Yup.number()
+      .typeError('Value is invalid.')
+      .transform(transformNumber)
+      .min(10, 'The minimum amount to bridge is 10.')
```

Comment:
Is this a requirement by @pendulum-chain/product or where does it come from?

Reply:
I am not sure where I took that number from. @prayagd we want to set a minimum for bridging ?

Reply:
@gonzamontiel i dont think so there is any minimum for bridging